addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp

```diff
@@ -0,0 +1,563 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Units
+//		* CUP Weapons
+//		* FIR AWS(AirWeaponSystem)
+//		* USM Uniforms
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {
+		"CUP_U_B_USMC_FROG1_WMARPAT",
+		"CUP_U_B_USMC_FROG2_WMARPAT",
+		"CUP_U_B_USMC_FROG3_WMARPAT",
+		"CUP_U_B_USMC_FROG4_WMARPAT"
+	};
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_Empty",
+		"CUP_V_B_MTV"
+	};
+	backpack[] = {};
+	headgear[] = {};
+	goggles[] = {"default"};
+	hmd[] = {};
+	// Leave empty to remove all. "Default" > leave original item.
+	faces[] = {};
+	// Leave empty to not change faces.
+	insignias[] = {};
+	// Leave empty to not change insignias
+
+	// All randomized
+	primaryWeapon[] = {};
+	scope[] = {};
+	bipod[] = {};
+	attachment[] = {};
+	silencer[] = {};
+	// Leave empty to remove all. "Default" for primaryWeapon > leave original weapon.
+
+	// Only *Weapons[] arrays are randomized
+	secondaryWeapon[] = {};
+	secondaryAttachments[] = {};
+	sidearmWeapon[] = {};
+	sidearmAttachments[] = {};
+	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
+
+	// These are added to the uniform or vest
+	magazines[] = {};
+	items[] = { MEDICAL_R ,"ACE_Flashlight_XL50","ACE_SpraypaintGreen"};
+	// These are added directly into their respective slots
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {
+		"CUP_H_USMC_MICH2000_WDL",
+		"CUP_H_USMC_MICH2000_COMM_WDL",
+		"CUP_H_USMC_MICH2000_DEF_WDL",
+		"CUP_H_USMC_MICH2000_DEF_ESS_WDL",
+		"CUP_H_USMC_MICH2000_ESS_COMM_WDL"
+	};
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_Rifleman",
+		"CUP_V_B_Eagle_SPC_Patrol",
+		"CUP_V_B_Eagle_SPC_Scout",
+		"CUP_V_B_MTV_Pouches",
+		"CUP_V_B_MTV_LegPouch"
+	};
+	primaryWeapon[] = {
+		"CUP_arifle_M16A4_Grip",
+		"CUP_arifle_M16A4_Base"
+	};
+	scope[] = {
+		"cup_optic_eotech553_black",
+		"cup_optic_compm4",
+		"cup_optic_compm2_low",
+		"cup_optic_compm2_black"
+	};
+	attachment[] = {
+		"cup_acc_anpeq_2_flashlight_black_l",
+		"cup_acc_anpeq_2_black_top"
+	};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("HandGrenade"),
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	primaryWeapon[] = {"CUP_arifle_M16A4_GL"};
+	attachment[] = {"cup_acc_anpeq_2_grey"};
+	vest[] = {"CUP_V_B_Eagle_SPC_GL"};
+	backPack[] = {"CUP_B_AssaultPack_Coyote"};
+	magazines[] += {
+		LIST_16("CUP_1Rnd_HEDP_M203"),

```

Comment:
Thats a lot of damage

Reply:
Lol, I actually kinda agree. I've toned it down to 11 rounds.